dev-python/sortedcontainers/sortedcontainers-2.1.0.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python2_7 python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="Python library to sort collections and containers"
+HOMEPAGE="http://www.grantjenks.com/docs/sortedcontainers/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
```

Comment:
The PyPI container seems to lack the test files so let's use the GitHub release tarball instead, please.